### PR TITLE
feat(charts): initial Helm chart

### PR DESCRIPTION
**Pull request by sebastiangaiser**
Closes https://github.com/strimzi/kafka-access-operator/issues/25

### CODE REVIEW

File: charts/kafka-access-operator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+# TODO
+appVersion: "0.1.0"
+description: "Strimzi Kafka access operator"
```
Comment (scholzj) [inline charts/kafka-access-operator/Chart.yaml:4]:
```suggestion
description: "Strimzi Kafka Access Operator"
```

File: charts/kafka-access-operator/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+name: kafka-access-operator
```
Comment (scholzj) [inline charts/kafka-access-operator/Chart.yaml:5]:
```suggestion
name: strimzi-access-operator
```

**Review by scholzj — COMMENTED** [review]
Thanks for the PR. I left some nits. Also, could you please move it to the `packaging` folder? As `packaging/helm-charts/...` similarly to as we have it for example [here](https://github.com/strimzi/drain-cleaner/tree/main/packaging)? We normally store the _work in progress things_ in `packaging/` and when released it is moved to the root directory.

**Comment by sebastiangaiser** [comment]
Done :D

**Review by scholzj — APPROVED** [review]
Thanks. LGTM. Any reason why to keep it Draft?

**Comment by sebastiangaiser** [comment]
Done. I think it's very limited yet but good to go :D

**Comment by katheris** [comment]
Thanks @sebastiangaiser. LGTM, as you say we can flesh it out a bit more once we confirm the first release version and image etc.

**Pull request merged**